cephfs/file_ops_test.go

```diff
@@ -0,0 +1,54 @@
+//go:build !nautilus && ceph_preview
+// +build !nautilus,ceph_preview
+
+package cephfs
+
+import (
+	"syscall"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMkNod(t *testing.T) {
+	mount := fsConnect(t)
+	defer fsDisconnect(t, mount)
+
+	file1 := "/file1"
+	mode1 := uint16(syscall.S_IFIFO | syscall.S_IRUSR | syscall.S_IWUSR)
+	err := mount.MkNod(file1, mode1, 0)
+	assert.NoError(t, err)
+
+	file2 := "/file2"
+	mode2 := uint16(syscall.S_IFCHR)
+	err = mount.MkNod(file2, mode2, 89)
+	assert.NoError(t, err)
+
+	file3 := "/file3"
+	mode3 := uint16(syscall.S_IFBLK)
+	err = mount.MkNod(file3, mode3, 129)
+	assert.NoError(t, err)
+
+	defer func() {
+		assert.NoError(t, mount.Unlink(file1))
+		assert.NoError(t, mount.Unlink(file2))
+		assert.NoError(t, mount.Unlink(file3))
+	}()
+
+	sx, err := mount.Statx(file1, StatxBasicStats, 0)
+	assert.Equal(t, sx.Mode&mode1, mode1)
+
+	sx, err = mount.Statx(file2, StatxBasicStats, 0)
+	assert.Equal(t, sx.Mode&mode2, mode2)
```

Comment:
Not a big deal but in future we may rearrange the arguments to `assert.Equal()` in such a way that it matches the [docs](https://pkg.go.dev/github.com/stretchr/testify/assert#Equal)? Second argument is the expected object.

Similarly for other occurrences.

Reply:
> Sure, I will update it in a follow-up PR for a new API.

Just for future new APIs 😛 I don't want you to change the current and already existing assertions.

Reply:
Haha, sure. I was talking about only updating the assertion in this API. But, I will let it be as it is :)

Reply:
NVM, taken care off in this PR itself :p